py/selenium/webdriver/chrome/webdriver.py

```diff
@@ -76,6 +76,43 @@ def launch_app(self, id):
         """Launches Chrome app specified by id."""
         return self.execute("launchApp", {'id': id})
 
+    def get_network_conditions(self):
+        """
+        Gets Chrome network emulation settings.
+
+        Returns:
+            A dict. For example:
+
+            {'sessionId': 'a5aa6326a9c4c8daaec9cdb3ce0e2f74',
+             'value': {
+                'latency': 4,
+                'download_throughput': 2,
+                'upload_throughput': 2,
+                'offline': False},
+              'status': 0}
+
+        """
+        return self.execute("getNetworkConditions")
+
+    def set_network_conditions(self, network_conditions):
+        """
+        Sets Chrome network emulation settings.
+
+        Sample usage:
+
+        driver.set_network_conditions({
+            'offline': False,
+            'latency': 5,  # additional latency (ms)
+            'download_throughput': 500 * 1024,  # maximal aggregated throughput
+            'upload_throughput': 500 * 1024  # maximal aggregated throughput
+        })
+
+        Note: 'throughput' can be used to set both (for download and upload).
+        """
+        return self.execute("setNetworkConditions", {
+            'network_conditions': network_conditions
+        })
```

Comment:
We may want these as named parameters and then build the dict ourself. @AutomatedTester, @davehunt opinions?